daprdocs/content/en/developing-applications/building-blocks/bindings/bindings-overview.md

```diff
@@ -20,9 +20,13 @@ For example, with bindings, your microservice can respond to incoming Twilio/SMS
 - Adding or configuring a third-party Twilio SDK
 - Worrying about polling from Twilio (or using WebSockets, etc.)
 
-{{% alert title="Note" color="primary" %}}
+<img src="/images/binding-overview.png" width=1000 alt="Diagram showing bindings">
+
+In the above diagram:
+- The input binding triggers your application using the endpoint name (which matches your input binding component name) and the method. 
```

Comment:
```suggestion
- The input binding triggers a method on your application.
```

Reply:
Can we change this sentence above "For example, with bindings, your microservice can respond to incoming Twilio/SMS messages without:"
to
"For example, with bindings, your application can respond to incoming Twilio/SMS messages without:"
prefer to use application here